### ISSUE TITLE
[textdiffuser-2] where to set the loss type during training?

### ISSUE THREAD

**Issue by shuyueW1991** [microsoft/unilm#1563]
hello, I am trying to use textdiffuser-2. When I am in the stage of M1's layout planning training, I found that there is no flag for setting the type of loss, whereas in the paper the author said such training requires loss type being cross-entropy. How come to set this configuration?  and by the way, how can I monitor the training loss curve like the one in readme.md?

**Comment by JingyeChen** [comment]
Thanks for your attention to TextDiffuser-2. Please dive into the Transformer Trainer code. The default loss is the cross entropy.

**Comment by shuyueW1991** [comment]
> Thanks for your attention to TextDiffuser-2. Please dive into the Transformer Trainer code. The default loss is the cross entropy.

thanks for you tips. i thought i missed some cricial argumental code haha.
